SonarQube.Bootstrapper/App.config

```diff
@@ -1,30 +1,33 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
-    <configSections>
-        <sectionGroup name="applicationSettings" type="System.Configuration.ApplicationSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" >
-            <section name="SonarQube.Bootstrapper.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
-        </sectionGroup>
-    </configSections>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
-    </startup>
-    <applicationSettings>
-        <SonarQube.Bootstrapper.Properties.Settings>
-            <setting name="PreProcessExe" serializeAs="String">
-                <value>SonarQube.MSBuild.PreProcessor.exe</value>
-            </setting>
-            <setting name="PostProcessExe" serializeAs="String">
-                <value>SonarQube.MSBuild.PostProcessor.exe</value>
-            </setting>
-            <setting name="PreProcessorTimeoutInMs" serializeAs="String">
-                <value>300000</value>
-            </setting>
-            <setting name="PostProcessorTimeoutInMs" serializeAs="String">
-                <value>3600000</value>
-            </setting>
-            <setting name="SonarQubeUrl" serializeAs="String">
-                <value />
-            </setting>
-        </SonarQube.Bootstrapper.Properties.Settings>
-    </applicationSettings>
+  <configSections>
+    <sectionGroup name="applicationSettings" type="System.Configuration.ApplicationSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" >
+      <section name="SonarQube.Bootstrapper.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
+    </sectionGroup>
+  </configSections>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
+  </startup>
+  <applicationSettings>
+    <SonarQube.Bootstrapper.Properties.Settings>
+      <setting name="PreProcessExe" serializeAs="String">
+        <value>SonarQube.MSBuild.PreProcessor.exe</value>
+      </setting>
+      <setting name="PostProcessExe" serializeAs="String">
+        <value>SonarQube.MSBuild.PostProcessor.exe</value>
+      </setting>
+      <setting name="PreProcessorTimeoutInMs" serializeAs="String">
+        <value>300000</value>
+      </setting>
+      <setting name="PostProcessorTimeoutInMs" serializeAs="String">
+        <value>3600000</value>
+      </setting>
+      <setting name="SonarQubeUrl" serializeAs="String">
+        <value />
+      </setting>
+      <setting name="SupportedBootstrapperVersionsXml" serializeAs="String">
```

Comment:
Why is the name of the file configurable? Are there any circumstances in which we'd want it to be changed on the client?